tests/test_csc.py

```diff
@@ -24,7 +24,7 @@ async def test_start(self):
             simulation_mode=1,
         ):
             await self.remote.cmd_start.set_start(
-                settingsToApply="homeoffice_pandemic_test.yaml"
+                settingsToApply="initial_test_config.yaml"
```

Comment:
You may want to have a specific set of configuration for testing. See https://github.com/lsst-ts/ts_ATDome/tree/develop/tests/data/config

Reply:
I will do this in the ticket dedicated to tests